HealthcareBase/Service/HospitalResourcesService/EquipmentService/EquipmentService.cs

```diff
@@ -41,11 +33,7 @@ public IEnumerable<EquipmentUnit> GetAll()
 
         public EquipmentUnit Create(EquipmentUnit equipmentUnit)
         {
-            if (equipmentUnit is null)
-                throw new BadRequestException();
-            if (!equipmentTypeRepository.Repository.ExistsByID(equipmentUnit.EquipmentType.GetKey()))
-                equipmentUnit.EquipmentType = equipmentTypeRepository.Repository.Create(equipmentUnit.EquipmentType);
-            return equipmentUnitRepository.Repository.Create(equipmentUnit);
+            return new EquipmentUnit();
```

Comment:
Why this method doesn't call repo, and why returns dummy object?

Reply:
Was trying something and forgot to implement the method after. Fixed